src/AzureStack/Subscriptions/Subscriptions.Tests/SessionRecords/SubscriptionsManagementClient/TestListRootOffers.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
"Root" offers is a misnomer. These are offers provided by the default delegated provider subscription. A.k.a admin.

Reply:
Fixing